storyboarding/scripting/compound_commands.rst

```diff
@@ -12,6 +12,405 @@ Loop
+
+I prepared some animations to use for running, tripping and getting up (actually the hardest part).
+
+.. image:: img/compound_commands/running.gif
+  :scale: 100%
+  :alt: Running character
+  :align: left
+
+.. image:: img/compound_commands/falling.gif
+  :scale: 100%
+  :alt: Falling character
+  :align: left
+
+.. image:: img/compound_commands/gettingUp.gif
+  :scale: 100%
+  :alt: Character getting up.
+  :align: left
+
+Let's start by putting the `Pass`-layer into place. While the gamestate is `Pass` the running-animation is displayed.
+
+.. code-block:: c
+  :linenos:
+  :caption: Character running in Pass-state
+  :name: RunningSimple
+
+  Animation,Background,Centre,"sb\running\running.png",320,240,240,150,LoopForever
+   F,0,0,,1
+
+Simple as that. We have to make another addition for the case that the `Passing` event is triggered.
+
+When this happens, Aoba is supposed to get up first before she starts running again. This means we have to fade the animation out for the process of getting up.
+
+.. code-block:: c
+  :linenos:
+  :caption: Character running in Pass-state with transition
+  :name: RunningComplete
+
+  Animation,Pass,Centre,"sb\running\running.png",320,240,6,150,LoopForever
+   F,0,0,,1
+   T,Passing,0,90000
+    F,0,0,,0
+    F,0,1000,,1
+
+The value of 1000 is arbitrary, it has to be set to the actual duration of getting up.
+
+Now the same is done for the `Fail`-layer with the `Failing` trigger. This time it is a sprite, not an animation as Aoba is just lying down.
+
+.. code-block:: c
+  :linenos:
+  :caption: Character lying on the ground
+  :name: Tripped
+
+  Sprite,Fail,Centre,"sb\falling\fallen.png",280,240
+   F,0,0,,1
+   T,Failing,0,90000
+    F,0,0,,0
+    F,0,2000,,1
+
+All that is left is putting the actual animations for falling and getting up in.
+
+.. code-block:: c
+  :linenos:
+  :caption: Character falling
+  :name: Tripping
+
+  Animation,Fail,Centre,"sb\falling\falling.png",280,240,9,150,LoopOnce
+   T,Failing,0,90000
+    F,0,0,,1
+    F,0,1000,,0
+
+.. code-block:: c
+  :linenos:
+  :caption: Character getting up
+  :name: GettingUp
+
+  Animation,Pass,Centre,"sb\gettingup\gettingup.png",320,240,8,150,LoopOnce
+   T,Passing,0,90000
+    F,0,0,,1
+    F,0,2000,,0
+
+Oh hey, that is easy, isn't it?
+
+Yes, too easy to actually work. Otherwise this would be the coolest interactive storyboard in 29 lines ever.
+
+The problem with using animations here is that they run independently from the trigger:
+
+  - When using `LoopOnce` as the loop-option they will work fine on the first trigger but show only the last animation frame on consecutive triggers.
+  - When using `LoopForever` the animation will work fine on the first trigger but start and end on the wrong frame.
+
+
+The slightly annoying but in this case bearable workaround is animating by hand which means creating a Sprite for every frame of the animation and fade it in and out with the delay according to its position within the animation.
```

Comment:
Be careful with the run-ons.

"In this case, the slightly annoying but bearable workaround is animating by hand, creating a sprite for every frame..."

Reply:
Fixed.